Dev

LeonardG1993 opened the pull request:
First PR to Main:
- Fully functioning app
- Need to refactor controllers as they use sessions
- Need to update some unit tests
- Currently using local DB for testing

acarasimon96 committed:
Begin work on ReviewController tests

CameronMSeibel committed:
Merge branch 'dev' of github.com:Revature-Sophia-ECommerce-1376/e-commerce-backend into cameron-seibel

SaberSams committed:
Update application.yml

CameronMSeibel committed:
Merge pull request #35 from Revature-Sophia-ECommerce-1376/damion-shirkey

added logging on ReviewService

hectororozco98 committed:
Merge pull request #36 from Revature-Sophia-ECommerce-1376/user-profile

User profile

acarasimon96 committed:
Finish success test cases for ReviewController GetMapping methods

SaberSams committed:
Update ProductController.java

bmpmd committed:
removed bin folder

CameronMSeibel committed:
Merge branch 'dev' of github.com:Revature-Sophia-ECommerce-1376/e-commerce-backend into cameron-seibel

janderson38 committed:
Merge branch 'exceptions-logging' of https://github.com/Revature-Sophia-ECommerce-1376/e-commerce-backend into jen

bmpmd committed:
Merge branch 'dev' into product-search

janderson38 committed:
added exceptions to product service

ktumlinson committed:
Merge pull request #34 from Revature-Sophia-ECommerce-1376/xin

input validation for create-update-product feature

janderson38 committed:
fixed typo on logger in Product Service

janderson38 committed:
Merge pull request #37 from Revature-Sophia-ECommerce-1376/jen

Jen

ktumli committed:
Merge conflict fix

ktumli committed:
slight change

ktumli committed:
slight conflict with yml fixed

ktumli committed:
Removed code smell

ktumli committed:
Attempting to get rid of the bug by doing something with the file.delete() boolean

CameronMSeibel committed:
Update data.sql to not contain null values for purchase

ktumlinson committed:
Merge pull request #38 from Revature-Sophia-ECommerce-1376/Kyle

Finished the upload image backend

CameronMSeibel committed:
Merge branch 'exceptions-logging' of github.com:Revature-Sophia-ECommerce-1376/e-commerce-backend into cameron-seibel

CameronMSeibel committed:
Merge pull request #39 from Revature-Sophia-ECommerce-1376/cameron-seibel

Cameron seibel

SamuelLiew committed:
Merge pull request #23 from Revature-Sophia-ECommerce-1376/product-search

Product search backend

SamuelLiew committed:
test

acarasimon96 committed:
Implement remaining ReviewController success test cases

acarasimon96 committed:
Code cleanup

- Moved request builders to their own local vars for readability
- Added MAPPING_ROOT constant to minimize potential bugs due to typos
and duplication

CameronMSeibel committed:
Revise exception handling for purchase service & controller

CameronMSeibel committed:
Add int constrcutor to Not Found exceptions to reduce duplicate literals

CameronMSeibel committed:
Revise review service to use int constructors for not found exceptions

CameronMSeibel committed:
Merge pull request #40 from Revature-Sophia-ECommerce-1376/cameron-seibel

Cameron seibel

JustDamion committed:
Merge branch 'dev' of https://github.com/Revature-Sophia-ECommerce-1376/e-commerce-backend into damion-shirkey

JustDamion committed:
added console logging (LoggingAdvice), added custom exceptions to UserService

JustDamion committed:
merged with exceptions-logging

acarasimon96 committed:
Begin work on ProductController tests

- Set up initial class structure
- Implement success test case for getInventory method

janderson38 committed:
Added AddressNotFoundException and added exceptions to the address service

JustDamion committed:
fixed minor bugs in LoggingAdvice and UserService

CameronMSeibel committed:
Merge pull request #42 from Revature-Sophia-ECommerce-1376/damion-shirkey

Added logger + exceptions

CameronMSeibel committed:
Merge pull request #43 from Revature-Sophia-ECommerce-1376/jen

Jen

CameronMSeibel committed:
Merge pull request #41 from Revature-Sophia-ECommerce-1376/testing

Add ReviewController unit tests

ktumli committed:
Final walk

ktumlinson committed:
Merge pull request #44 from Revature-Sophia-ECommerce-1376/Kyle

Final walk

ktumli committed:
Removed code smell

ktumlinson committed:
Merge pull request #46 from Revature-Sophia-ECommerce-1376/Kyle

Removed code smell

ktumli committed:
Fixing merge conflicts with dev

ktumlinson committed:
Merge pull request #47 from Revature-Sophia-ECommerce-1376/Kyle

Fixed merge conflicts

ktumli committed:
Fixed code smell

ktumlinson committed:
Merge pull request #48 from Revature-Sophia-ECommerce-1376/Kyle

Fixed code smell

janderson38 committed:
Added custom exception for findUsersAddresses and fixed Address update exceptions

CameronMSeibel committed:
Merge pull request #49 from Revature-Sophia-ECommerce-1376/jen

Added custom exception for findUsersAddresses and fixed Address updat…

ktumlinson committed:
Merge pull request #45 from Revature-Sophia-ECommerce-1376/create-update-products

Create update products is complete

acarasimon96 committed:
Add remaining ProductController GetMapping method tests

acarasimon96 committed:
Implement insertAndUpdate success test cases

This also adds a EqualsAndHashCode annotation into the Product model,
which is necessary for the tests to pass

vickeryRev committed:
fixed code smell in productinfo :  removed unsued import

vickeryRev committed:
Merge branch 'debug' of https://github.com/Revature-Sophia-ECommerce-1376/e-commerce-backend into debug

hectororozco98 committed:
Fixed code smells in AuthService and Aspect

vickeryRev committed:
fixed code smell in authorized + authRestriction : changec const loggedin to all uppercase to follow convention

hectororozco98 committed:
Merge branch 'debug' into hector

hectororozco98 committed:
Merge pull request #50 from Revature-Sophia-ECommerce-1376/hector

Fixed code smells in AuthService and Aspect

hectororozco98 committed:
Merge pull request #51 from Revature-Sophia-ECommerce-1376/debug

Debug

arhughes11 committed:
created template for readme.md

arhughes11 committed:
Merge branch 'dev' into documentation
merging in dev

TTaylorPro committed:
Removed a security vulnerability from ProductController's upsert method

hectororozco98 committed:
Merge branch 'debug' into travis

hectororozco98 committed:
Merge pull request #52 from Revature-Sophia-ECommerce-1376/travis

Removed a security vulnerability from ProductController's upsert method

SamuelLiew committed:
test

SamuelLiew committed:
Merge branch 'dev' of https://github.com/Revature-Sophia-ECommerce-1376/e-commerce-backend into dev

SamuelLiew committed:
huge changes

LeonardG1993 committed:
Merge pull request #53 from Revature-Sophia-ECommerce-1376/Security

Security

acarasimon96 committed:
Fix compile errors in service tests

acarasimon96 committed:
Write tests for newer ProductService methods

filterByRating tests are not yet implemented because I'm having a hard
time understanding its code in ProductRepository. With that said,
ProductService still achieves 91% test coverage.

acarasimon96 committed:
Add findByEmail unit test

jmt0494 committed:
set up spring security with Oauth2. Implemented jwt auth and endpoint protection

acarasimon96 committed:
Add PurchaseService tests

This also adds the EqualsAndHashCode annotation to the Purchase model
class for the tests to pass

acarasimon96 committed:
Add AddressService tests

This also adds EqualsAndHashCode annotation to the Address model class
to be able to make the tests pass

addAddress method test currently fails because it's missing code that
finds an existing Address record whose ID matches the update request.

hectororozco98 committed:
Merge pull request #56 from Revature-Sophia-ECommerce-1376/dev

Dev to debug pull

hectororozco98 committed:
Added some comments to PurchaseController and Service

LeonardG1993 committed:
Merge pull request #55 from Revature-Sophia-ECommerce-1376/testing

Update and add new service tests

acarasimon96 commented:
> * Currently using local DB for testing

I don't think the unit tests use any form of database. Rather, I only used dummy model instances and mocks to write them

arhughes11 committed:
quick edit to README.md

arhughes11 committed:
changing README adding images

sophiagavrila commented on src/main/java/com/revature/controllers/ReviewController.java:
```diff
@@ -0,0 +1,141 @@
+
+@RestController
+@RequestMapping("/api/review")
+@CrossOrigin(origins = {"http://localhost:4200", "http://localhost:3000"}, allowCredentials = "true")
```
Change this to `@CrossOrigin(origins="*", allowedHeaders="*")` to allow frontend in prod environment to reach this controller without throwing a CORS error

sophiagavrila commented on src/main/java/com/revature/controllers/UserController.java:
```diff
@@ -0,0 +1,68 @@
+
+@RestController
+@RequestMapping("/api/users")
+@CrossOrigin(origins = {"http://localhost:4200", "http://localhost:3000"}, allowCredentials = "true")
```
Change this to `@CrossOrigin(origins="*", allowedHeaders="*") `to allow frontend in prod environment to reach this controller without throwing a CORS error

sophiagavrila commented on src/main/java/com/revature/controllers/PurchaseController.java:
```diff
@@ -0,0 +1,64 @@
+
+@RestController
+@RequestMapping("api/purchases")
+@CrossOrigin(origins = {"http://localhost:4200", "http://localhost:3000"}, allowCredentials = "true")
```
Change this to `@CrossOrigin(origins="*", allowedHeaders="*")` to allow frontend in prod environment to reach this controller without throwing a CORS error

sophiagavrila commented on src/main/java/com/revature/controllers/ProductController.java:
```diff
@@ -1,50 +1,117 @@
 @RestController
-@RequestMapping("/api/product")
+@RequestMapping("/api/product" )
 @CrossOrigin(origins = {"http://localhost:4200", "http://localhost:3000"}, allowCredentials = "true")
```
Change this to `@CrossOrigin(origins="*", allowedHeaders="*")` to allow frontend in prod environment to reach this controller without throwing a CORS error

sophiagavrila commented on src/main/java/com/revature/controllers/AddressController.java:
```diff
@@ -0,0 +1,66 @@
+
+@RestController
+@RequestMapping("/api/addresses")
+@CrossOrigin(origins = {"http://localhost:4200", "http://localhost:3000"}, allowCredentials = "true")
```
Change this to `@CrossOrigin(origins="*", allowedHeaders="*")` to allow frontend in prod environment to reach this controller without throwing a CORS error

sophiagavrila commented:
Resolve code smells - do not use `System.out.println(...)` where you could instead use a Logger.

Code smell report here https://sonarcloud.io/project/issues?id=Revature-Sophia-ECommerce-1376_e-commerce-backend&pullRequest=54&resolved=false&types=CODE_SMELL

sophiagavrila commented:
Add Jacoco Plugin to the `pom.xml` to generate test coverage report. It's a simple add - paste from here and follow instructions outlined here https://docs.sonarcloud.io/enriching/test-coverage/java-test-coverage/#:~:text=Add%20coverage%20in,the%20project%20root.

hectororozco98 committed:
Merge pull request #57 from Revature-Sophia-ECommerce-1376/hector

Added some comments to PurchaseController and Service

hectororozco98 committed:
Merge pull request #58 from Revature-Sophia-ECommerce-1376/comment

Comment

hectororozco98 committed:
Merge pull request #59 from Revature-Sophia-ECommerce-1376/dev

Dev

SamuelLiew committed:
Merge branch 'Security' of https://github.com/Revature-Sophia-ECommerce-1376/e-commerce-backend into dev

Swicky97 committed:
Fix code smells in back end

Swicky97 committed:
Merge pull request #60 from Revature-Sophia-ECommerce-1376/chase

Fix code smells in back end

CameronMSeibel committed:
Merge changes from dev

CameronMSeibel committed:
Merge pull request #61 from Revature-Sophia-ECommerce-1376/cmseibel

Cmseibel

CameronMSeibel committed:
oopsie

CameronMSeibel committed:
Merge pull request #62 from Revature-Sophia-ECommerce-1376/cmseibel

oopsie

SaberSams committed:
Updated docker-compose to point to port 5000

spring-boot has been set internally to host to port 5000 to better align with industry norms so docker-compose has to be updated

SaberSams committed:
Update build.sh

build script updated to use docker-compose as application requires an SQL server

CameronMSeibel committed:
Play with logging advice, properly name rest controller

acarasimon96 committed:
Add coverage profile to pom.xml

hectororozco98 committed:
Working on code smells

JustDamion committed:
Update LoggingAdvice.java

CameronMSeibel committed:
Change names in RestExceptionHandler

CameronMSeibel committed:
Merge branch 'exceptions-logging' of github.com:Revature-Sophia-ECommerce-1376/e-commerce-backend into cmseibel

hectororozco98 committed:
Merge pull request #64 from Revature-Sophia-ECommerce-1376/hector

Working on code smells

CameronMSeibel committed:
Merge pull request #65 from Revature-Sophia-ECommerce-1376/cmseibel

Cmseibel

acarasimon96 committed:
Merge pull request #63 from Revature-Sophia-ECommerce-1376/testing

Integrate JaCoCo test coverage to SonarCloud

sophiagavrila committed:
added sonar properties to pom.xml for coverage report

sophiagavrila committed:
Create build.yml

acarasimon96 committed:
Skip failing ReviewController test case

acarasimon96 committed:
Skip failing AddressService testUpdate test

acarasimon96 committed:
Remove ECommerceApplicationTests

acarasimon96 committed:
Merge pull request #66 from Revature-Sophia-ECommerce-1376/testing

Skip failing test cases

JustDamion committed:
added custom exceptions to StorageService

JustDamion committed:
merged with logging-exceptions

CameronMSeibel committed:
Merge pull request #68 from Revature-Sophia-ECommerce-1376/damion-shirkey

Added custom exceptions to StorageService

JustDamion committed:
Merge branch 'testing' of https://github.com/Revature-Sophia-ECommerce-1376/e-commerce-backend into damion-shirkey

CameronMSeibel committed:
Remove superfluous sysout

hectororozco98 committed:
Merge pull request #69 from Revature-Sophia-ECommerce-1376/hector

Hector

hectororozco98 committed:
Merge pull request #70 from Revature-Sophia-ECommerce-1376/user-profile

User profile

CameronMSeibel committed:
Add comments to review controller and service

CameronMSeibel committed:
Merge from upstream

CameronMSeibel committed:
Merge pull request #71 from Revature-Sophia-ECommerce-1376/cmseibel

Cmseibel

CameronMSeibel committed:
Make minor fixes to tests to not fail the build

CameronMSeibel committed:
Merge pull request #73 from Revature-Sophia-ECommerce-1376/cmseibel

Make minor fixes to tests to not fail the build

acarasimon96 committed:
Fix ReviewController.getReviews empty list test case

Apparently, it's intentional for the "api/review" endpoint to return a 200
OK status code instead of 204 even if there are no review records stored
in the database

CameronMSeibel committed:
Refactor address service 'n' controller so they work a little bit better

CameronMSeibel committed:
Fix tests for that wacky address service we all know and love

CameronMSeibel committed:
Merge pull request #75 from Revature-Sophia-ECommerce-1376/cmseibel

Cmseibel

ktumli committed:
test

CameronMSeibel committed:
Try explicit constructor definition

CameronMSeibel committed:
Merge pull request #76 from Revature-Sophia-ECommerce-1376/cmseibel

Try explicit constructor definition

CameronMSeibel committed:
Merge branch 'testing' of github.com:Revature-Sophia-ECommerce-1376/e-commerce-backend into cmseibel

ktumli committed:
working on the addressController

acarasimon96 committed:
Merge branch 'dev' into testing

CameronMSeibel committed:
Merge branch 'dev' of github.com:Revature-Sophia-ECommerce-1376/e-commerce-backend into cmseibel

ktumli committed:
Conflict fix

JustDamion committed:
added testing for UserController

acarasimon96 committed:
Remove unused ID field from AddressRequest

This little unexplained change caused compile errors that prevented
GitHub Actions to build the project successfully

CameronMSeibel committed:
Merge branch 'testing' of github.com:Revature-Sophia-ECommerce-1376/e-commerce-backend into cmseibel

acarasimon96 committed:
Fix code smell

acarasimon96 committed:
Merge pull request #74 from Revature-Sophia-ECommerce-1376/testing

Fix ReviewController.getReviews empty list test case

CameronMSeibel committed:
Keep the all args constructor, I guess

CameronMSeibel committed:
Update service tests to fall in line with new exceptions

acarasimon96 committed:
Add PurchaseController success test cases

CameronMSeibel committed:
Simply return optional product in ProductService.findByID

CameronMSeibel committed:
Modify Product Service tests to match up to revisions

CameronMSeibel committed:
Update purchase service tests to reflect revisions to service

CameronMSeibel committed:
Merge pull request #77 from Revature-Sophia-ECommerce-1376/cmseibel

Cmseibel

CameronMSeibel committed:
Merge branch 'dev' of github.com:Revature-Sophia-ECommerce-1376/e-commerce-backend into cmseibel

CameronMSeibel committed:
Merge pull request #78 from Revature-Sophia-ECommerce-1376/cmseibel

Cmseibel

CameronMSeibel committed:
Now we don't have an ID I guess

CameronMSeibel committed:
Merge pull request #79 from Revature-Sophia-ECommerce-1376/cmseibel

Now we don't have an ID I guess

LeonardG1993 committed:
Merge pull request #72 from Revature-Sophia-ECommerce-1376/exceptions-logging

Exceptions logging

acarasimon96 committed:
Add PurchaseController failure test cases

Code in PurchaseController.getPurchasesByOwner and
PurchaseService.findByOwner for the "user not found" test case need
to be implemented by the purchases feature team

acarasimon96 committed:
Exclude product reviews and purchases from hashcode

This causes a StackOverflowError when running tests due to circular
dependency issues

acarasimon96 committed:
Add ProductController.purchase success test case

SamuelLiew committed:
potential update

JustDamion committed:
Merge branch 'testing' of https://github.com/Revature-Sophia-ECommerce-1376/e-commerce-backend into damion-shirkey

JustDamion committed:
fixed issue with UserControllerTest

hectororozco98 committed:
Removed RequiredArgsConstructor annotation

hectororozco98 committed:
Merge pull request #81 from Revature-Sophia-ECommerce-1376/hector

Removed RequiredArgsConstructor annotation

LeonardG1993 committed:
Merge pull request #80 from Revature-Sophia-ECommerce-1376/debug

Debug

acarasimon96 committed:
Enable filterByRating unit test

acarasimon96 committed:
Merge branch 'dev' into testing

acarasimon96 committed:
Merge pull request #82 from Revature-Sophia-ECommerce-1376/testing

Add ProductController tests

hectororozco98 committed:
Merge pull request #83 from Revature-Sophia-ECommerce-1376/dev

Updating debug branch

acarasimon96 committed:
Return 404 error if User/Product is not found

acarasimon96 committed:
Merge pull request #84 from Revature-Sophia-ECommerce-1376/testing

Add PurchaseController tests

hectororozco98 committed:
Took care of code smells in the back end

hectororozco98 committed:
Merge pull request #85 from Revature-Sophia-ECommerce-1376/hector

Took care of code smells in the back end

JustDamion committed:
added custom exception to findByEmail in UserService, refactored testUpdate_Success in UserControllerTest

acarasimon96 committed:
Implement AuthController tests

JustDamion committed:
Merge branch 'testing' of https://github.com/Revature-Sophia-ECommerce-1376/e-commerce-backend into damion-shirkey

acarasimon96 committed:
Assert session contents in login and logout tests

JustDamion committed:
fixed all current tests in UserControllerTest

acarasimon96 committed:
Add missing EqualsAndHashCode annotation to User model

The existing poor implementation of User.hashCode caused
AuthController.register tests to fail, so replacing the hashCode method
with a proper one and its missing equals method made the tests pass.

Apparently, the missing EqualsAndHashCode annotation is a recurring
theme in our model classes, causing all sorts of unexpected bugs in
other parts of the project.

acarasimon96 committed:
Merge pull request #86 from Revature-Sophia-ECommerce-1376/testing

Add AuthController tests

JustDamion committed:
Merge branch 'testing' of https://github.com/Revature-Sophia-ECommerce-1376/e-commerce-backend into damion-shirkey

JustDamion committed:
Merge branch 'testing' of https://github.com/Revature-Sophia-ECommerce-1376/e-commerce-backend into damion-shirkey

JustDamion committed:
removed access modifiers from UserControllerTest to fix code smells

JustDamion committed:
Merge pull request #87 from Revature-Sophia-ECommerce-1376/damion-shirkey

Added UserControllerTest

SamuelLiew committed:
added roles

SamuelLiew committed:
fix merge conflicts

CameronMSeibel committed:
Add tests for 100 coverage in ReviewService

ktumli committed:
Merge branch 'advice-aspect-tests' of https://github.com/Revature-Sophia-ECommerce-1376/e-commerce-backend into Kyle
getting updated version

CameronMSeibel committed:
Add tests to hit 100 coverage in PurchaseService

CameronMSeibel committed:
Add tests to hit 100 coverage in UserService

CameronMSeibel committed:
Hit 100 coverage in AddressService

CameronMSeibel committed:
Improve coverage in ProductService to 96

ktumli committed:
Address controller tests done(other than not logged in)

ktumli committed:
Fixed code smell

ktumli committed:
Added Comments

ktumli committed:
Made requested changes

acarasimon96 committed:
Merge pull request #89 from Revature-Sophia-ECommerce-1376/Kyle

Added AddressController tests

SamuelLiew committed:
small changes

hectororozco98 committed:
Added EqualsAndHashCode annotation and Excludes to sets

hectororozco98 committed:
Merge pull request #90 from Revature-Sophia-ECommerce-1376/hector

Added EqualsAndHashCode annotation and Excludes to sets

gaschindler committed:
refactored purchase controller to not utilize http session

vickeryRev committed:
fixed a code smell in models/products

vickeryRev committed:
fixed a code smell in productService logging, added a method that takes and int and returns a string.
this si called each time that warning is needed (i really dont know if this is an improvment)

vickeryRev committed:
Merge branch 'debug' of https://github.com/Revature-Sophia-ECommerce-1376/e-commerce-backend into debug

vickeryRev committed:
fixed a code smell in storageService, removed duplicate <String> decleration

gaschindler committed:
refactored review controller to purge session use

gaschindler committed:
removed currentSchema arg from datasource url

SamuelLiew committed:
struggling with user and address

gaschindler committed:
removed unnecessary imports from controllers

LeonardG1993 committed:
Merge pull request #91 from Revature-Sophia-ECommerce-1376/SecurityTotallyMVP

Security implemented, controllers refactored

hectororozco98 committed:
Merge branch 'dev' into debug

ktumli committed:
Fixed issues with merge conflict

hectororozco98 committed:
Fixed build error

hectororozco98 committed:
Merge pull request #94 from Revature-Sophia-ECommerce-1376/hector

Fixed build error

ktumli committed:
One more conflict

acarasimon96 committed:
Merge branch 'dev' into testing

# Conflicts:
#	src/main/java/com/revature/config/StorageConfig.java
#	src/main/java/com/revature/controllers/ProductController.java
#	src/main/java/com/revature/controllers/UserController.java
#	src/main/java/com/revature/services/UserService.java
#	src/main/resources/application.yml

CameronMSeibel committed:
Merge changes from dev

acarasimon96 committed:
Fix test compile errors caused by recent addition of Spring Security

acarasimon96 committed:
Exclude timestamp fields from EqualsAndHashCode

Those fields being included in equals and hashCode methods were causing
PotentialStubbingProblem exceptions in ProductServiceTest and
ReviewServiceTest

CameronMSeibel committed:
Resolve merge conflicts from upstream

CameronMSeibel committed:
Merge changes from upstream

CameronMSeibel committed:
Missed a change, oopsie!

CameronMSeibel committed:
Update fail messages in service tests

SamuelLiew committed:
fixed address

JustDamion committed:
added testing for getters/setters for all models

SamuelLiew committed:
fixed address and user

SamuelLiew committed:
Merge branch 'dev' of https://github.com/Revature-Sophia-ECommerce-1376/e-commerce-backend into SecurityTotallyMVP

LeonardG1993 committed:
Merge pull request #96 from Revature-Sophia-ECommerce-1376/SecurityTotallyMVP

Security totally mvp

ktumli committed:
Conflict fixes

JustDamion committed:
added testing for getters/setters for all dtos

CameronMSeibel committed:
Write basic exceptions  testing to make sure they are what they are I guess

acarasimon96 committed:
Merge branch 'dev' into testing

ktumli committed:
Fixed addressControllerTest

ktumli committed:
Fixed issue with TestConfig

arhughes11 committed:
added documentation for controllers+service

JustDamion committed:
Merge branch 'testing' of https://github.com/Revature-Sophia-ECommerce-1376/e-commerce-backend into damion-shirkey

JustDamion committed:
added hashcode() and equals() testing for all models and dtos

CameronMSeibel committed:
Modify security config to not interfere with test config

ktumli committed:
Fixed ProductController and PurchaseController tests

CameronMSeibel committed:
Write security config for testing purposes

CameronMSeibel committed:
Implement test security in review and user controller tests

CameronMSeibel committed:
Merge branch 'testing' of github.com:Revature-Sophia-ECommerce-1376/e-commerce-backend into cmseibel

JustDamion committed:
Merge pull request #95 from Revature-Sophia-ECommerce-1376/damion-shirkey

added testing for getters/setters and hashcode/equals for all models and dtos

CameronMSeibel committed:
Merge from upstream

acarasimon96 committed:
Merge pull request #97 from Revature-Sophia-ECommerce-1376/Kyle

Fixed AddressControllerTest, ProductControllerTest, PurchasesControllerTest, and added TestConfig file

CameronMSeibel committed:
Merge branch 'testing' of github.com:Revature-Sophia-ECommerce-1376/e-commerce-backend into cmseibel

CameronMSeibel committed:
Replace with old test security config with TestConfig from upstream

CameronMSeibel committed:
Move away from Map.of since we can't use it :^(

CameronMSeibel committed:
Modified findUsersAddresses test to use assertFalse instead of assertTrue

CameronMSeibel committed:
Merge pull request #99 from Revature-Sophia-ECommerce-1376/cmseibel

Cmseibel

acarasimon96 committed:
Merge branch 'dev' into testing

acarasimon96 committed:
Reduce verbosity of unit test logs

LeonardG1993 committed:
Merge pull request #98 from Revature-Sophia-ECommerce-1376/testing

Big testing update

hectororozco98 committed:
Merge branch 'dev' into debug

hectororozco98 committed:
Updating hector branch

hectororozco98 committed:
Merge pull request #100 from Revature-Sophia-ECommerce-1376/hector

Updating debug branch

hectororozco98 committed:
Merge pull request #93 from Revature-Sophia-ECommerce-1376/debug

Pushing debug into dev
